tensorflow/core/kernels/mkl_concat_op.cc

```diff
@@ -306,12 +315,18 @@ class MklConcatFwdPrimitive : public MklPrimitive {
  private:
   // Primitive reuse context for concat Fwd op
   struct ConcatFwdContext {
+#ifndef ENABLE_MKLDNN_V1
     std::vector<mkldnn::memory::primitive_desc> src_pd;
     std::vector<std::shared_ptr<mkldnn::memory::primitive_desc>> src_pd_shdptr;
     std::shared_ptr<mkldnn::memory::primitive_desc> dst_pd;
+#endif  // ENABLE_MKLDNN_V1
 
-    // MKL-DNN memory
+// MKL-DNN memory
+#ifdef ENABLE_MKLDNN_V1
+    std::vector<mkldnn::memory> data_mem;
+#else
     std::vector<mkldnn::primitive::at> data_mem;
+#endif  // ENABLE_MKLDNN_V1
```

Comment:
Nit, since this is `#ifndef`
```suggestion
#endif  // !ENABLE_MKLDNN_V1
```

Reply:
thanks. will do